### PR TITLE
[BREAKING] Webdriver Bidi support

### PR DESCRIPTION
**Pull request by kblok**
First green of a navigation test.

## Breaking changes

* `ITarget.TargetId` was removed.

### CODE REVIEW

File: lib/PuppeteerSharp/PuppeteerSharp.csproj [hunk 1]
```diff
@@ -26,6 +26,7 @@
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <LangVersion>12</LangVersion>
     <AnalysisLevel>8.0-Recommended</AnalysisLevel>
+    <NoWarn>CS8002</NoWarn>
```
Comment (kblok) [inline lib/PuppeteerSharp/PuppeteerSharp.csproj:29]: TODO: Remove once webdriver bidi is signed.

File: lib/PuppeteerSharp/PuppeteerSharp.csproj [hunk 2]
```diff
@@ -56,4 +55,7 @@
   <ItemGroup>
     <EmbeddedResource Include="Injected\injected.js" />
   </ItemGroup>
+  <ItemGroup>
```
Comment (Meir017) [inline lib/PuppeteerSharp/PuppeteerSharp.csproj:58]: should this be a PackageReference to https://github.com/webdriverbidi-net/webdriverbidi-net ?

Comment (kblok) [reply lib/PuppeteerSharp/PuppeteerSharp.csproj:58]: I'm just backing up my work when I switch between branches.
We will use the real package.

**Comment by kblok** [comment]
@Meir017 @jnyrup is open for review if you want to spend sometime reading the bidi code 😂 ?
There is quite a long way to feature parity, but hey! We have 36 passing tests.

File: lib/PuppeteerSharp/PuppeteerSharp.csproj [hunk 3]
```diff
@@ -43,6 +43,7 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.Extensions.Logging" Version="8.0.0" />
+    <PackageReference Include="WebDriverBiDi" Version="0.0.9" />
```
Comment (jnyrup) [inline lib/PuppeteerSharp/PuppeteerSharp.csproj:46]: Would be convenient if `WebDriverBiDi` had a .NET target without any nuget dependencies.

Comment (jnyrup) [reply lib/PuppeteerSharp/PuppeteerSharp.csproj:46]: FYI: `WebDriverBiDi` has a newer release with the added TFMs
image

Comment (kblok) [reply lib/PuppeteerSharp/PuppeteerSharp.csproj:46]: Yes. I'm working on a new PR that needs that version 😉

**Pull request merged**